src/firefly/js/visualize/ui/FitsHeaderView.jsx

```diff
@@ -24,17 +24,18 @@ const popupPanelResizableStyle = {
     height: 400,
     minHeight: 300,
     resize: 'both',
-    overflow: 'auto',
+    overflow: 'hidden',
     position: 'relative'
 };
 
 
 //define the display style for the file size and pixel information and the table in the same div
-const tableAndTitleInfoStyle = {width: '100%', height: 'calc(100% - 40px)'};
+const tableAndTitleInfoStyle = {width: '100%', height: 'calc(100% - 40px)', display: 'flex'};
 
 //define the table style only in the table div
-const tableStyle = {width: '100%', height: 'calc(100% - 30px)',  overflow: 'auto', flexGrow: 1, display: 'flex'};
+const tableStyle = {paddingLeft:5,paddingRight:5, width: '100%', height: 'calc(100% - 60px)',  overflow: 'hidden', flexGrow: 1, display: 'flex'};
```

Comment:
FYI:  not a big deal, since it's not very visible.  But, by adding padding while keeping 100% width just pushes the whole table to the right.  So, if you look closely, you will see that the table is chopped off by 5px to the right.  Had parent's overflow was not 'hidden', a scrollbar would appears.
You can fix this by using calc to subtract the paddings.  But, a better way to do this is to use box-sizing: border-box.  This takes paddings and borders into the '100%' calculation.